dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1582,7 +1593,7 @@ def _get_min(
         subset_properties: dict,
     ) -> None:
         min_value = df_series.min()
-        self.min = min_value if not self.min else min(self.min, min_value)
+        self.min = float(min_value) if not self.min else float(min(self.min, min_value))
```

Comment:
Cast added to adhere to type specification in numeric stats mixin attribute initialization (avoids setting to np.float64)